src/Intro/Intro.jsx

```diff
@@ -43,108 +44,106 @@ const styles = () => ({
 	},
 });
 
-const Intro = props => (
-	<div>
-		<section id="intro" className={props.classes.section}>
-			<Hidden smUp>
-				<ResponsiveImage path="/images/skystars.jpg" asBackgroundImage />
-			</Hidden>
-			<Hidden only="xs">
-				<video autoPlay muted loop className={props.classes.fullVideo}>
-					<source
-						src={`${process.env.PUBLIC_URL}/2719001.mp4`}
-						type="video/mp4"
-					/>
-				</video>
-			</Hidden>
-			<Container fullHeight center>
-				<div className="row justify-content-center">
-					<div className="col-12 col-md-6">
-						<Card className={props.classes.card}>
-							<CardContent>
-								<div className="row justify-content-center">
-									<div className="col-10 col-md-6 mb-3 mt-2">
-										<img
-											src={`${
-												process.env.PUBLIC_URL
-											}/images/logo-codestar-by-ordina.svg`}
-											alt="Codestar powered by Ordina Logo"
-										/>
-									</div>
-								</div>
-								<Typography
-									variant="subheading"
-									gutterBottom
-									className={props.classes.introText}
-								>
-									The digital world offers endless possibilities. The challenges
-									are often complex. We develop state-of-the-art software that’s
-									simple to use. Agile and productive, using the latest
-									techniques. We program with our hearts and with our minds, for
-									organisations looking to take the next step.
+@translate(['intro'], { wait: true })
+class Intro extends Component {
+	render() {
+		const props = this.props;
+		const { t } = this.props;
```

Comment:
I would rather a more self documenting expression like `translate` or `i18n`

Destructuring and renaming can be done as `{  t: translate } = this.props`
Applies to rest of the files

Reply:
`t()` is what is used and recommended all around `react-i18next`'s documentation.
So I guess we have to make a collective choice here and use that decision onward

Reply:
That was my motivation too, that it is conform the docs.